Makefile

```diff
@@ -1,39 +1,13 @@
 include $(TOPDIR)/rules.mk
-
-PKG_NAME:=luci-app-chinadns-ng
 PKG_VERSION:=1.0
 PKG_RELEASE:=4
 
-include $(INCLUDE_DIR)/package.mk
-
-define Package/$(PKG_NAME)
-	SECTION:=luci
-	CATEGORY:=LuCI
-	SUBMENU:=3. Applications
-	TITLE:=LuCI support for chinadns-ng
-	LUCI_DEPENDS:=+chinadns-ng
-endef
-
-define Build/Prepare
-	$(foreach po,$(wildcard ${CURDIR}/luasrc/i18n/*.po), \
-		po2lmo $(po) ${CURDIR}/luasrc/i18n/$(patsubst %.po,%.lmo,$(notdir $(po)));)
-endef
-
-define Build/Configure
-endef
+LUCI_TITLE:=LuCI support for chinadns-ng
+LUCI_DEPENDS:=+chinadns-ng
 
-define Build/Compile
-endef
+PKG_MAINTAINER:==pexcn <i@pexcn.me>
+PKG_LICENSE:=GPL-3.0-or-later
```

Comment:
Please remove `PKG_MAINTAINER` and `PKG_LICENSE`, this branch is not maintain by me, see more: https://github.com/pexcn/openwrt-chinadns-ng/issues/12